tool/tctl/common/token_command.go

```diff
@@ -288,16 +281,20 @@ func (c *TokenCommand) List(client auth.ClientI) error {
 	return nil
 }
 
-// calculateCAPin returns the SPKI pin for the local cluster.
-func calculateCAPin(client auth.ClientI) (string, error) {
-	localCA, err := client.GetClusterCACert()
+// calculateCAPins returns the SPKI pins for the local cluster.
+func calculateCAPins(client auth.ClientI) ([]string, error) {
+	caCerts, err := client.GetClusterCACerts()
```

Comment:
If the client is talking to an older auth server, this won't work. Can we do a fallback to the old API method here?

Reply:
I ended up updating the client `GetClusterCACerts` method to fall back to the old endpoint on error

Reply:
I've not also added back to old API method for compat with a slightly out of date `teleport.e` so I can get CI to pass. I tested this with both new client/old server and new server/old client, it works for both.